shell/platform/windows/flutter_window.h

```diff
@@ -181,6 +331,70 @@ class FlutterWindow : public Window, public WindowBindingHandler {
   // proper application lifecycle state can be updated once the view is set.
   bool restored_ = false;
   bool focused_ = false;
+  // Activates tracking for a "mouse leave" event.
+  void TrackMouseLeaveEvent(HWND hwnd);
+
+  // Stores new width and height and calls |OnResize| to notify inheritors
+  void HandleResize(UINT width, UINT height);
+
+  // Retrieves a class instance pointer for |window|
+  static FlutterWindow* GetThisFromHandle(HWND const window) noexcept;
+
+  // Updates the cached scroll_offset_multiplier_ value based off OS settings.
+  void UpdateScrollOffsetMultiplier();
+
+  int current_dpi_ = 0;
+  int current_width_ = 0;
+  int current_height_ = 0;
```

Comment:
We should order members in sections. I'll check the style guide but my recollection is: statics, regular methods, fields. Just make sure we keep fields ordered the same as constructor field initialisation to avoid undefined behaviour.